.github/workflows/npm.yml

```
@@ -0,0 +1,36 @@
+name: NPM
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "contracts/**"
+      - "package.json"
+      - "package-lock.json"
+  workflow_dispatch:
+
+jobs:
+  npm-compile-publish-contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+          cache: "npm"
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Bump up package version
+        id: npm-version-bump
+        uses: keep-network/npm-version-bump@v2
+        with:
+          environment: dev
```

This project contains only common contracts that we will use in other projects. We're not expecting to deploy them within this repo, so there won't be packages tagged like `development`, `ropsten`, `mainnet`. Maybe we should tag it as `pre`?

```suggestion
          environment: pre
```

---

See 196ed307ec3419845aa84c2dd9cd00d9d19f00fc.